doc/sphinx/cython/scripts.rst

```diff
@@ -0,0 +1,13 @@
+.. _sec-converter-documentation:
+
+Python Converter Modules
+========================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 1
+
+   cti_conversion
+   ctml_conversion
+   yaml_conversion
```

Comment:
Can a page for `cantera.ck2yaml` be created that just uses `.. automodule` as well, or is there some problem that prevents that from working?

Reply:
👍 ... It absolutely works.